pep-XXXX.rst

```diff
@@ -0,0 +1,258 @@
+PEP: XXXX
+Title: A Platform Tag for Portable Linux Built Distributions
+Version: $Revision$
+Last-Modified: $Date$
+Author: Robert T. McGibbon <rmcgibbo@gmail.com>, Nathaniel J. Smith <njs@pobox.com>
+Status: Draft
+Type: Process
+Content-Type: text/x-rst
+Created: 01-Jan-2016
+Post-History: 30-Aug-2002
+
+
+Abstract
+========
+
+This PEP proposes the creation of a new platform tag for Python package built
+distributions, such as wheels, called ``manylinux_1_{x86_64,i386}`` with
```

Comment:
Okay, here's a pure style/taste nitpick: should it be `manylinux_1` or `manylinux1`? I prefer `manylinux1` just for consistency with the look of most of the other wheel tags `py2`, `cp35`, `win32`, `i386`, `amd64` (OTOH there's also `macosx_10_9` and `x86_64`, but I think they're the only ones, so I guess the pattern is that underscores before numbers are only used when multiple numbers go together?). Doesn't matter too much, but probably worth a few seconds of thought.